### ISSUE TITLE
Create ticket from task ?

### ISSUE THREAD

**Issue by Sismic** [InfotelGLPI/tasklists#42]
Good morning,

can you think it will be a good idea if we can convert (or copy) a task to a ticket ?

**Comment by derry43** [comment]
I also think this is a great idea and linking the ticket and task at the same time would be a great way to go with this. 

We use tasklist to implement the GTD methodology and being able to move a task out to become a ticket or create a ticket would be a great way to move a item on to the correct place to process it. Being able to choose between creating a ticket from a task and closing the task or creating a ticket, keeping the task open and linking to the task would be ideal. In the first case, the task might be “Create a ticket for Jane”, so no need to keep the task open after ticket is created. In the second case, the task might be “help Jane with find a file” but the ticket might only ask the server team to “check if file is in the backups”, we may still have work to do on the task outside of the ticket.

Also, the opposite idea, being able to create a tasklist task from the ticket tasklist tab would also be worth considering (just as you can create a planning task from a ticket. Sometimes you want an unplanned task (rather than a planned task - planning) from a ticket that might eventually get closed, so that you can track other spin-off work associated with but not part of the fix for the ticket.

**Comment by paulz1** [comment]
Still nothing on this issue ?

It could be great to have a possibility to create task directly from ticket (or show tickets in tasklist Kanban).
Sure, there is already Kanban view for the tickets. But we need something different.
Kanban view for the ticket use tickets status. But in our case we want to have some different organisation for the ticket (in parallel to the standard status) and tasklist kanban could be very helpful for this.
